src/corehost/build.proj

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Any new OS that we are trying to boot-strap. The native builds need to be able to build without any managed dependencies working yet like msbuild.